### PR TITLE
39 finish login

### LINKED ISSUE #39: Finish login
Connect **login** frontend and backend part

### PR DESCRIPTION
**Pull request by maryanchuk04**
Add Services for login user in frontend
Fix some error on backend

### CODE REVIEW

File: StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
 import React, { useState } from "react";
 
-function PasswordTextField({ className = "", placeholder, required, setDisabled, handleChange, secondValue = "", validate }) {
+function PasswordTextField({ className = "", placeholder, required, setDisabled, handleChange, secondValue = "", validate = null }) {
 	const [hidden, setHidden] = useState(false);
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx:4]: This file i think we can remove after merge

File: StudCity.Frontend/src/UI/Fields/TextField.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import React from "react";
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/Fields/TextField.jsx:2]: identical situation

File: StudCity.Frontend/src/services/authenticateService.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { BaseService } from "./baseService";
+import { TokenService } from "./tokenService";
+
+export class AuthenticateService {
+	// Inject alert provider service.
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/services/authenticateService.js:5]: We add alert service in redux and can dispatch action in methods

**Review by Serhiy-dotcom — APPROVED** [review]
Looks good!

**Review by Vova-Romaniuk — APPROVED** [review]

**Review by maria2908 — APPROVED** [review]
Great job!

**Pull request merged**